src/lucky_record/where.cr

```diff
@@ -112,4 +116,24 @@ module LuckyRecord::Where
       Ilike.new(@column, @value)
     end
   end
+
+  class In < SqlClause
+    def operator
+      "= ANY"
+    end
+
+    def negated : NotIn
+      NotIn.new(@column, @value)
+    end
+  end
+
+  class NotIn < SqlClause
+    def operator
+      "!= ALL"
+    end
+
+    def negated : In
```

Comment:
We could add two expectations in the `where_spec.cr` to make sure the negated part is as expected.

Reply:
Oh I didn't see those specs, thanks!